src/site/content/en/blog/css-masking/index.md

```diff
@@ -0,0 +1,196 @@
+---
+title: How to use CSS masking
+description: |
+  CSS masking gives you the option of using an image as a mask layer.
+  This means that use can use an image, an SVG, or a gradient as your mask,
+  to create interested effects without an image editor.
+authors:
+  - rachelandrew
+date: 2020-09-02
+hero: hero.jpg
```

Comment:
Can you poke around on unsplash.com and find some other kind of metaphor to use here? I want to avoid reminding people of covid19.